experiments/tf_trainer/keras_gru_attention/run.hp.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Is run.hp for hyper parameter tuning? Let's rename this file to make that more clear. Also a one line comment at the top would be great. I'm planning on going back to add similar comments to the other run scripts.

Reply:
done